client/src/pages/HomePage.js

```diff
@@ -1,12 +1,9 @@
 import "./HomePage.css";
-import TeaserImage from "../assets/42460be23cc1d9abd1a45a44b16d1a8c.jpg";
 
 export default function HomePage() {
   return (
     <div className="App__Page App__HomePage">
-      <h1>Welcome to Bibliophilia - your personal Bookshelf</h1>
-      <img src={TeaserImage} className="TeaserImage" />
-      <h3>... developed by Chris during the neuefische bootcamp!</h3>
+      Home - perhaps place for an dashboard?!
```

Comment:
If this is not helping the user (he can't use it) then this has no purpose in this pr.